packages/wallet/backend/src/utils/jwk.ts

```diff
@@ -0,0 +1,21 @@
+import { createPublicKey, generateKeyPairSync } from 'crypto'
+
+export const generateJWK = () => {
```

Comment:
you could also `pnpm i @interledger/http-signature-utils` to get the `generateJwk` function here

Reply:
![image](https://github.com/interledger/testnet/assets/101926344/b8401ad5-0e72-4d89-a815-c13135d8d68e)

It seems not compatible...

Reply:
I think what is not compatible here is the `rafikiClient.createRafikiPaymentPointerKey` function. You would just need to  update how you pass in the `generatedJwk`

Reply:
If you don't want to go that route, we should remove "@interledger/http-signature-utils" in that case